.changeset/silver-games-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+Added a set of default prometheus metrics around scaffolding
```

Comment:
Could we include a list of what metrics will be available? Also, some details about the labels would be nice.

Might be worth having a link to the metrics guide in the contrib folder as I think some people won't have that setup.

For the Vale error I'm wondering if you need to capitalize "prometheus" this to make Vale happy or to add it to the list of words it knows about.

Reply:
Addressed, please review. Also I'm not in love with the metric names if you think there's something clearer. mostly I thought of 'task' as a 'Template' and 'step' as a 'Task' but bootstrapping isnt where I spend most of my time and i just tried to follow the class's convention

Reply:
There's something odd going on with the formatting, somethings have bullets that shouldn't so it's hard to read. Here's what I'm seeing:

![image](https://user-images.githubusercontent.com/67169551/190640330-0a9a607b-46aa-445f-a8b9-3c6e03d91266.png)

Reply:
Looks good now 👍